package-lock.json

```diff
@@ -15,7 +15,7 @@
     "@firebase/auth": {
```

Comment:
It is weird to have changes in the lock file without corresponding changes in `package.json`. Please revert.